### PR TITLE
Upgrade django to 3 and wagtail to 2.12

### LINKED ISSUE #6702: Step 1. Upgrade Django ato Django 3, and Wagtail to 2.12
## Acceptance criteria

**Note** This will need to be 2 PRs. 

- [ ] Upgrade Wagtail to 2.12.* and upgrade Django to 3.0.*. Keep django-modeltranslation and wagtail-modeltranslation. We'll remove these later when upgrading to Wagtail 2.13 and Django 3.2.

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #6702 

**Note**: We need to upgrade to Django 3.1 and then 3.2 later along with Wagtail 2.13. I believe these are in #6705 

**Note**: #6720 needs to be applied before this PR can be accepted.

### CODE REVIEW

File: docs/stack.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ We use [wagtail-modeltranslations](https://github.com/infoportugal/wagtail-model
 
 #### A/B testing
 
-We use [wagtail-experiments](https://github.com/torchbox/wagtail-experiments) for CMS-based A/B testing. Please see its documentation for more information.
+We currently don't have any A/B testing going on. We used to use [wagtail-experiments](https://github.com/torchbox/wagtail-experiments) but we're going to switch over to [wagtail-ab-testing](https://github.com/torchbox/wagtail-ab-testing)
```
Comment (KalobTaulien) [inline docs/stack.md:33]: We don't think wagtail-experiments are being used. It's an old app that doesn't support Django 3 so I removed it. We'll be adding in wagtail-ab-testing in a coming sprint at some point as well, which would have removed this dependency anyway

File: network-api/networkapi/wagtailcustomization/__init__.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+from django.templatetags.static import static
+from django.utils.html import format_html
+
+from wagtail.core import hooks
+
+
+@hooks.register("insert_global_admin_css", order=100)
+def global_admin_css():
+    """Add /static/css/admin.css to the admin."""
+    return format_html('<link rel="stylesheet" href="{}">', static("css/admin.css"))
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailcustomization/__init__.py:10]: This file had different line endings. But the only thing that changed was there `static` is coming from.

File: requirements.in [hunk 3]
```diff
@@ -20,15 +20,14 @@ python-slugify
 requests
 social-auth-app-django
 wagtail-airtable
-wagtail==2.11.7
+wagtail==2.12.4
```
Comment (KalobTaulien) [inline requirements.in:23]: We can't upgrade to Wagtail 2.13 and Django 3.2 until wagtail-modeltranslation supports Django 3.2 and Wagtail 2.13. But we'll be removing modeltranslation in #6705

**Comment by Pomax** [comment]
This looks good as far as I can tell, but let's run that collection fix first =)

**Review by Pomax — APPROVED** [review]
this looks good to me. I've run the `clean_collections` command on both staging and prod, so we should be able to land this.

Tomorrow, because it's too late today =)

**Pull request merged**